kuttl-test.yaml

```diff
@@ -11,7 +11,7 @@ testDirs:
 - tests/kuttl/cgu/
 - tests/kuttl/ibgu/
 crdDir: ./config/crd/bases/
-timeout: 25
+timeout: 40
```

Comment:
Can this be set on a per test basis?

Reply:
it can be set for assert files, but we need it in the step files.